pokemon-react-app/src/redux/reducers/pokemonListReducer.js

```diff
@@ -0,0 +1,32 @@
+import { SET_POKEMONS, POKEMONS_LOADING, FILTER_POKEMONS } from "../types";
+
+const initialState = {
+  pokemons: [],
+  filteredPokemons: [],
+  filterStatus: false,
```

Comment:
Pretty good logic with Redux. I would drop using filterStatus and have in filteredPokemons all the time what is displayed in the UI.